CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+CHANGES IN VERSION 1.9.1 (from 1.9.0)
+===================================
+
+Appium 1.9.1 is a patch release
```

Comment:
If we stop supporting Xcode 8 in the next release, is it helpful to mention it?
=> https://github.com/appium/WebDriverAgent/pull/116